Base/home/anon/.shellrc

```diff
@@ -1,7 +0,0 @@
-#!/bin/sh
-
-# IFS controls what $(...) (inline evaluate) would split its captured
-# string with. the default is \x0a (i.e. newline).
-IFS="\x0a"
-
-export HISTCONTROL="ignoreboth"
```

Comment:
Why are you removing the shellrc file?

Reply:
I created the fork about three weeks ago, the .shellrc file was not part of my fork :/

Reply:
you should rebase your changes on latest master then :)

Reply:
Ohh, ok, thanks for the tip, I'll try to do that.